src/App.tsx

```diff
@@ -1,24 +1,194 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos, createTodo, destroyTodo } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterStatus } from './types/FilterStatus';
+import { ErrorBlock } from './components/ErrorBlock/ErrorBlock';
 
-const USER_ID = 0;
+const USER_ID = 10266;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string>('');
```

Comment:
it's extra, BC TS already knows about this types - you put an initial value
```suggestion
  const [error, setError] = useState('');
```